processing/src/main/java/io/druid/query/aggregation/Aggregator.java

```diff
@@ -51,6 +51,11 @@ default double getDouble()
     return (double) getFloat();
   }
 
+  default boolean isNull()
```

Comment:
can we add a comment here what this function is for and when should be implemented?